deployment/source-only-mode.md

```diff
@@ -0,0 +1,76 @@
+# Source Only Mode
+
+Since v1.18.0, Fluentd can launch with source-only mode.
+(Not supported on Windows)
+
+* In this mode, only input plugins run.
+* During this mode, the events are stored in a temporary file buffer.
+* Sending `SIGWINCH` to the supervisor cancels this mode.
+  * Then, all plugins start to run, and the temporary file buffer starts to load.
+
+## How to launch Fluentd with source-only mode
+
+You can launch Fluentd with source-only mode in the following ways.
+
+* [Command Line Option - --with-source-only](command-line-option.md)
+* [System Configuration - with_source_only](system-config.md#with_source_only)
+
+## Temporary file buffer
+
+During source-only mode, the ingested events are stored in a temporary file buffer.
+After `SIGWINCH` is sent to the supervisor and this mode is canceled, this buffer begins to load.
+
+By default, the file buffer path is as follows.
+
+* `/tmp/fluent/source-only-buffer/{Unique ID for the Fluentd instance}`
```

Comment:
It might be better to clarify "What instance".
e.g supervisor, worker,  plugin and so on.

Reply:
It is a common ID for the supervisor and all workers belonging to the supervisor, but I can't explain it briefly...

Reply:
Hmm, so let's keep it as is.